### DISCUSSION TITLE
Adding youtube or blog resources

### DISCUSSION THREAD

**Discussion by amireshoon** [BlackIQ/Hello-World#88] [Ideas]
It's gonna be useful to add some youtube channels or some blogs URL for pushing the user to know more about language or if he/she wants to skill up him/her self skills have some resources.
Adding `resources.md` in the language folder and contributors can push resources into it.

**Comment by parsampsh** [comment]
See #90

**Reply by amireshoon** [reply]
Perfect. Thanks
